sebastien15/capstoneproject
# Ft create a new article

#### What does this PR do?
> this PR create and save article to mongodb
> This PR upload featured image for a single article and display them in postman

#### Description of Task to be completed
>The added 2more packages to process image namely: multer and sharp
> Added two files for multer configurations, imageProcessor midlleware
> I've created a router to save an article to mongodb and show the results in postman

#### How should this be manually tested?

- Clone this repo with `git clone git@github.com:sebastien15/myportifolio.git` 
- Checkout to `branch_name`
- npm install nodemon -g
- npm install
- npm start
- run localhost:3000/add-blog in post man make sure it is a post method

#### Any background context you want to provide?
>N/A
#### What are the relevant Trello story?
>https://trello.com/c/ZjoTIkX3
#### Screenshots (if appropriate)
>test in post man
![add blo](https://user-images.githubusercontent.com/41580903/93977700-2ca46f00-fd7b-11ea-8888-d4404ce13355.PNG)
